sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -606,15 +606,15 @@ object GpuOverrides extends Logging {
    * @return true if it is allowed else false
    */
   def isSupportedType(dataType: DataType,
-      allowNull: Boolean = false,
-      allowDecimal: Boolean = false,
-      allowBinary: Boolean = false,
+      allowNull: Boolean = true,
```

Comment:
It would not be a good choice to change the default values here in its definition.